lsstseries/analysis/stetsonj.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+
+def calc_stetson_J(flux, err, band, band_to_calc=None):
+    """Compute the stetsonJ statistic on data from one or several bands
+        Parameters
+        ----------
+        flux : `numpy.ndarray` (N,)
+            Array of flux/magnitude measurements
+        err : `numpy.ndarray` (N,)
+            Array of associated flux/magnitude errors
+        band : `numpy.ndarray` (N,)
+            Array of associated band labels
+        band_to_calc : `str` or `list` of `str`
+            Bands to calculate stetson J on. Single band descriptor, or list 
```

Comment:
Motivated by comment by Melissa - lets change this comment here to ``StetsonJ''.

Reply:
Done